providers/base/bin/prime_offload_tester.py

```diff
@@ -148,6 +166,60 @@ def check_offload(
         self.logger.info("  Couldn't find process [{}]".format(cmd))
         self.check_result = True
 
+    def _find_bdf(self, card_id: str):
+        """
+        Use the /sys/kernel/debug/dri/<card id>/name to get pci BDF.
+
+        :param card_id: card id shows in debugfs
+        """
+        filename = "/sys/kernel/debug/dri/{}/name".format(card_id)
+        with open(filename, "r") as f:
+            data_in_name = f.read()
+        return data_in_name.split()[1].split("=")[1]
+
+    def find_offload(self, cmd: str, timeout: int):
+        """
+        Find the card that the command is running on.
+
+        :param cmd: command that is running
+
+        :param timeout: timeout for command
+        """
+        directory = "/sys/kernel/debug/dri"
+
+        delay = timeout / 10
+
+        deadline = time.time() + timeout
+
+        cmd = cmd.split()
+
+        while time.time() < deadline:
+            time.sleep(delay)
+            card_path = self._if_file_contain(directory, "clients", cmd[0])
```

Comment:
Specify that cmd[0] is the name of the process you are looking for. (I think that's the case)